src/executables/utility.h

```diff
@@ -880,6 +882,7 @@ typedef struct _ha_config
 #define CHECK_TRANSACTION_SERVER  "Transaction-Server"
 
 #define COMMDB_SERVER_STOP      "-S"
+#define COMMDB_SINGLE_NODE_STOP "-n"
```

Comment:
Make the commdb command self-explanatory by using only the long version and naming it `single-node-stop`. See also the comment on commdb arguments.
```suggestion
#define COMMDB_SINGLE_NODE_STOP "--single-node-stop"
```